.github/workflows/pr.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       GO_VERSION: '1.20'
       KIND_VERSION: '0.20.0'
       K8S_VERSION: '1.26.x'
+      SKIP_PACT_TESTS: 'true'
```

Comment:
Not sure what you mean. If we remove that, it will run the Pact tests too. Do you mean you would like to move the login somewhere else or run the Pact tests with unit tests?

Reply:
I refactored tests and now it's not needed there.